src/main/java/fi/hsl/pulsar/mqtt/MessageProcessor.java

```diff
@@ -151,4 +156,18 @@ public boolean isMqttConnected() {
         }
         return false;
     }
+
+    public boolean lastMsgReceivedTimeHealthy() {
```

Comment:
rename as isProcessorStateHealthy() or just isHealthy()
(method name doesn't have to indicate how it is calculated, isProcessorStateHealthy better describes what it returns)

Reply:
@Divinare Since this is not the only health check method in MessageProcessor, it is nice to know what checks we are adding to healthServer (in Main) ?